engine/table/src/test/java/io/deephaven/engine/util/scripts/TestGroovyDeephavenSession.java

```diff
@@ -104,5 +124,486 @@ public void testUnloadedWildcardPackageImport() {
+                // quoting - ignore for now, they are not valid in the import statement context
+        };
+        for (String testCase : testCases) {
+            final String[] parts = testCase.split("\\|");
+            final String p1 = parts.length > 1 ? parts[1] : "";
+            assertEquals(p1, GroovyDeephavenSession.removeComments(parts[0]));
+            // System.out.println("Pass: \"" + parts[0] + "\" -> \"" + p1 + "\"");
+        }
+    }
+
+    // things to import for testIsValidImportString
+    @SuppressWarnings("unused")
+    public static class StaticClass {
+        public static int field;
+        public static int field1;
+        public static int field2;
+
+        public static void method() {}
+
+        public static void method1() {}
+
+        public static void method2() {}
+
+        public static class InnerStaticClass {
+            public static int field;
+            public static int field1;
+            public static int field2;
+
+            public static void method() {}
+
+            public static void method1() {}
+
+            public static void method2() {}
+
+            public static class InnerInnerStaticClass {
+                public static int field;
+                public static int field1;
+                public static int field2;
+
+                public static void method() {}
+
+                public static void method1() {}
+
+                public static void method2() {}
+            }
+        }
+    }
+
+    @SuppressWarnings("unused")
+    public int field; // testIsValidImportString
+
+    @SuppressWarnings("unused")
+    public void method() {}
+
+    ; // for testIsValidImportString
+
+    /**
+     * test the code that checks an import request for validity.
+     */
+    @Test
+    public void testIsValidImportString() throws IOException {
+        final String[] failTestCases = new String[] {
+                "import import io.deephaven.engine.util.TableTools", // two imports
+                "static import io.deephaven.engine.util.TableTools", // static before import
+                "import // io.deephaven.engine.util.TableTools", // invalid after stripped comments
+                "import io.deephaven.engine.util.TableTools.", // trailing .
+                "import io.deephaven.engine.util.TableTools.**", // invalid wildcard
+                "import io.deephaven.engine.util.TableTools.*.*", // doubled wildcard
+                "import io.deephaven.engine. util.TableTools; // has a space", // internal space
+                "import io.deephaven.engine.xxx.util.TableTools", // nonexistent package
+                "import io.deephaven.engine.util.TablexxxTools", // nonexistant classname
+                "import static io.deephaven.engine.util.scripts.TestGroovyDeephavenSession.StaticClassstatic ", // includes
+                                                                                                                // "static
```

Comment:
Should update these comments (copied with the tests from dhe) so they appear on one line